students/Chris_Kenyon/Lesson_2/PrintGrid.py

```diff
@@ -0,0 +1,66 @@
+###Lesson 2 making a program for creating grids of various sizes###
+
+#set values for simplicity
+p = "+"
+m = "-"
+s = " "
+v = "|"
+
+###Part 1: Brute force###
+#create a 2x2 grid of pre-specified size using print commands
+print(p + (m*4) + p + (m*4) + p)
+for i in range(4):
+	print(v + (s *4) + v + (s*4) + v)
+print(p + (m*4) + p + (m*4) + p)
+for i in range(4):
+	print(v + (s *4) + v + (s*4) + v)
+print(p + (m*4) + p + (m*4) + p)
+
+###Part 2: Making a basic function###
+#create a 2x2 grid of defined size using a function
+
+def print_grid(GridSize):
+	"""Create a 2x2 grid with with grid size determined from function argument"""
+	print(p + (m*GridSize) + p + (m*GridSize) + p)
+	for i in range(GridSize):
+		print(v + (s *GridSize) + v + (s*GridSize) + v)
+	print(p + (m*GridSize) + p + (m*GridSize) + p)
+	for i in range(GridSize):
+		print(v + (s *GridSize) + v + (s*GridSize) + v)
+	print(p + (m*GridSize) + p + (m*GridSize) + p)
+	
+###Part 3: Two Part Function###
+#create a grid with variable rows and columns of variable size
+
+def print_grid2(GridSize,RowCol):
+	"""Create a grid with with grid size and number of rows and columns determined from function arguments"""
+	if GridSize > 0 and RowCol > 0: #check for valid inputs
```

Comment:
per python style guid: `snake_case` for var names and function names

Reply:
I'm not sure what you mean by "snake_case" should I avoid the mix capital.  I could not find any reference to "snake_case" in the PEP8 or style guidelines on the website

Reply:
update: I looked up "snake case" on google and will use that going forward

Reply:
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names

this is `snake_case` and this is `camelCase` and this is `PascalCase` or `UpperCamelCase`

https://www.google.com/search?q=what+is+snake+case&oq=what+is+snake+case&aqs=chrome..69i57j0.1838j1j9&sourceid=chrome&ie=UTF-8

Reply:
👍